python/dftd4/pyscf.py

```diff
@@ -23,6 +23,7 @@
 try:
     from pyscf import lib, gto
     from pyscf.grad import rhf as rhf_grad
+    from pyscf.data.elements import charge as sym2chrg
```

Comment:
Can we use `gto.charge` here instead?

Reply:
That's actually the same function.
https://github.com/pyscf/pyscf/blob/bf0b1db22556a3c1b4c34426ea8627e636c1b096/pyscf/gto/__init__.py#L22
https://github.com/pyscf/pyscf/blob/bf0b1db22556a3c1b4c34426ea8627e636c1b096/pyscf/gto/mole.py#L49